catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/UserRepository.java

```diff
@@ -121,11 +121,13 @@ public User getByEmail(String email, Fields fields) throws IOException, ParseExc
 
   @Override
   public User setFields(User user, Fields fields) throws IOException, ParseException {
-    user.setProfile(fields.contains("profile") ? user.getProfile() : null);
-    user.setTeams(fields.contains("teams") ? getTeams(user) : null);
-    user.setRoles(fields.contains("roles") ? getRoles(user) : null);
-    user.setOwns(fields.contains("owns") ? getOwns(user) : null);
-    user.setFollows(fields.contains("follows") ? getFollows(user) : null);
+    if (fields != null) {
```

Comment:
I was under the impression that Fields is never null and we use Fields.EMPTY_FIELDS instead.  

Is there a place where this assumption is not valid?

Reply:
You are right. We don't pass null anywhere for Fields. Removed this null check in the latest patch.